static/src/js/util/url/collectionsEncoders.js

```diff
@@ -201,6 +221,15 @@ const decodedSelectedVariables = (pgParam) => {
   return variableIds.split('!')
 }
 
+const decodedSwodlrDownload = (pgParam) => {
+  const { swod: swodlrData } = pgParam
+  if (!swodlrData) {
```

Comment:
We need a test case for this else block (also it can be reduce to a single line)